src/ert/gui/model/snapshot.py

```diff
@@ -211,9 +212,25 @@ def _add_partial_snapshot(self, partial: PartialSnapshot, iter_: int):
                 if job.index:
                     job_node.data[ids.INDEX] = job.index
                 if job.current_memory_usage:
-                    job_node.data[ids.CURRENT_MEMORY_USAGE] = job.current_memory_usage
+                    job_node.data[ids.CURRENT_MEMORY_USAGE] = float(
+                        job.current_memory_usage
+                    )
+                    real_node.data[ids.CURRENT_MEMORY_USAGE] = float(
+                        job.current_memory_usage
+                    )
+                    self.root.data[ids.CURRENT_MEMORY_USAGE] = float(
+                        job.current_memory_usage
+                    )
                 if job.max_memory_usage:
                     job_node.data[ids.MAX_MEMORY_USAGE] = job.max_memory_usage
+                    real_node.data[ids.MAX_MEMORY_USAGE] = max(
+                        real_node.data.get(ids.MAX_MEMORY_USAGE, -1),
+                        float(job.max_memory_usage),
+                    )
+                    self.root.data[ids.MAX_MEMORY_USAGE] = max(
+                        self.root.data.get(ids.MAX_MEMORY_USAGE, -1),
+                        float(job.max_memory_usage),
+                    )
```

Comment:
Also here, and why didn't we cast for the first one?
```suggestion
                    max_memory_usage = float(job.max_memory_usage)
                    job_node.data[ids.MAX_MEMORY_USAGE] = max_memory_usage
                    real_node.data[ids.MAX_MEMORY_USAGE] = max(
                        real_node.data.get(ids.MAX_MEMORY_USAGE, -1), max_memory_usage)
                    self.root.data[ids.MAX_MEMORY_USAGE] = max(
                        self.root.data.get(ids.MAX_MEMORY_USAGE, -1), max_memory_usage)
```

Reply:
Good question, I probably missed it.
Good suggestion!